### DISCUSSION TITLE
Connect  wind farm to power system

### DISCUSSION THREAD

**Discussion by TanBendong** [CURENT/andes#152] [Q&A] [answered]
Dr. Cui,

I want to add hundreds of double-fed induction generators (DFIG) to the power system.  It seems there is no model of DFIG in the library. Is it possible to use ANDES to build DFIG with defined components, i.e., ResPitch and ResTprque? Thanks!

Best regards,
Bendong Tan

**Comment by cuihantao** [comment, marked as answer]
Hi,

It is possible to use ANDES to build DFIG models. You can refer to the model in Power System Analysis Toolbox or Milano's book "Power System Modelling and Scripting". 

The equations and limiters can be implemented straightforward.

**Comment by TanBendong** [comment]
Thanks！I think ANDES is similar to Simulink, which is able to connect all components to construct a system, such as DFIG? Is that right?

**Reply by cuihantao** [reply]
It's not exactly like SImulink. ANDES does not have a graphical user interface, but ANDES supports easy vectorization for large systems.

**Reply by cuihantao** [reply]
DFIG is a model and can be part of a large system.

**Reply by TanBendong** [reply]
Thanks for your detailed answer!  It is great software!

**Comment by cuihantao** [comment]
Thanks. Feel free to post more specific questions.

Regards,
Hantao Cui
